examples/player/simid_player.js

```diff
@@ -120,7 +127,14 @@ class SimidPlayer {
    * Plays a SIMID  creative once it has responded to the initialize ad message.
    */
   playAd() {
-    this.contentVideoElement_.pause();
+
+    // content video shouldn't be paused for non-linear ads
```

Comment:
Why not move all this logic down below where we have the other
if (this.isLinear_) statement?

Keep the pausing and playing stuff together.